src/main/java/com/sergiofah/model/Category.java

```diff
@@ -1,13 +1,17 @@
 package com.sergiofah.model;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
 
 import javax.persistence.*;
-import java.io.Serializable;
 import java.util.List;
 
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
 @Entity
 @Table(name = "category")
-public class Category implements Serializable {
-
+public class Category{
```

Comment:
The left brace must always be separated from the previous character by most Java code conventions and our convention.